### DISCUSSION TITLE
Pre compiled PyPI version of poselib

### DISCUSSION THREAD

**Discussion by iago-suarez** [cvg/glue-factory#20] [Ideas]
Hi guys, what do you think about proposing @vlarsson add poselib to PyPI with pre-computed manylinux binaries? This dependency consumes most of the installation time when we compile in extra mode!

**Comment by vlarsson** [comment]
Hi Iago,
@Parskatt has started (https://github.com/PoseLib/PoseLib/issues/72) setting up pypi for PoseLib and there is a version online. However it is currently broken for some systems and I would for now recommend just compiling master. But once it's fixed you should be able to switch to pypi.

**Reply by iago-suarez** [reply]
Hi,

That sounds very good, just let me know if you need any help with it 🤗

**Reply by Parskatt** [reply]
I'd love some help! I haven't had the time recently, but you can see the setup in #72

It needs to be merged with some recent changes, and better tested. Theres also the issue with linux builds sometimes failing, and mac builds having issues as well. The mac issues I think can be fixed quite easily.

**Reply by iago-suarez** [reply]
Alright, let's do the following, if @Parskatt you merge the new changes you mention, next I can take care of fixing some CI actions and doing some testing to assure that everything is working well :)
